src/port.js

```diff
@@ -232,6 +247,18 @@ export default class Port {
 				.resolve(handlerResult)
 				.then((val) => {
 					me.sendMessage(`res.${requestType}`, { id: w.id, val: val });
+				})
+				.catch((e) => {
+					const err = {
+						message: e.message,
+						name: e.name,
+						props: {}
+					};
+					for (let prop of Object.keys(e)) {
+						err.props[prop] = e[prop];
+					}
+
+					me.sendMessage(`res.${requestType}`, { id: w.id, err });
```

Comment:
Not doing this for the moment because of the way _messageType_/_subType_ is found. _resErr_ would be seen as a _res_, and then everything would go weird anyway.